django_dramatiq/admin.py

```diff
@@ -35,7 +37,16 @@ def eta(self, instance):
         return datetime.fromtimestamp(timestamp, tz=tz)
 
     def message_details(self, instance):
-        message_details = json.dumps(instance.message._asdict(), indent=4)
+        message_dict = instance.message._asdict()
+
+        # make sure we can still get a representation of the
+        # kwargs payload when a non json encoder is in use
+        kwargs_encoder = DjangoDramatiqConfig.select_encoder()
+        if not isinstance(kwargs_encoder, JSONEncoder):
+            for k,v in message_dict['kwargs'].items():
+                message_dict['kwargs'][k] = f"<{v}>"
+
+        message_details = json.dumps(message_dict, indent=4)
```

Comment:
That seems like a code that solves a different issue rather than #133. Was the second commit pushed by a mistake? It would be good to address different issues separately. 🙏